### PR TITLE
Add `lines-around-directive` rule

### PR DESCRIPTION
**Pull request by SimenB**
My vote is obviously for no space above, and a space below.

Thoughts? It's not autofixable, but I can send a PR for that later, _shouldn't_ be too difficult.

http://eslint.org/docs/rules/lines-around-directive

EDIT: PR for autofix: eslint/eslint#7217

### CODE REVIEW

**Comment by theneva** [comment]
> shouldn't

**Comment by sveisvei** [comment]
Sure 👍

File: packages/eslint-config-finn/index.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = {
         'comma-dangle': ['error', 'always-multiline'],
         'computed-property-spacing': 'error',
         'eqeqeq': ['error', 'smart'],
+        'lines-around-directive': ['error', { before: 'never', after: 'always' }],
```
Comment (SimenB) [inline packages/eslint-config-finn/index.js]: A bunch of examples in the eslint docs.
But then they'd have to be together. Do we use header comments though?

**Comment by SimenB** [comment]
I'll bump minimal eslint to get the autofix, merge this, and release `1.0.0` after that

**Pull request merged**